filestore/handlers.py

```diff
@@ -60,15 +60,18 @@ def __init__(self, fpath, template, filename, frame_per_point=1):
         self._image_sequence = pims.ImageSequence(self._filename)
 
     def __call__(self, point_number):
-        start, stop = point_number * self._fpp, (point_number + 1) * self._fpp
+        point_number = int(point_number)
+        start= point_number * self._fpp
+        stop = (point_number + 1) * self._fpp
         if stop > len(self._image_sequence):
             # if asking for an image past the end, make sure we have an up to
             # date list of the existing files
             self._image_sequence = pims.ImageSequence(self._filename)
-        if stop > len(self._image_sequence):
-            # if we _still_ don't have enough files, raise
-            raise IntegrityError("Seeking Frame {0} out of {1} frames.".format(
-                stop, len(self._image_sequence)))
+            if stop > len(self._image_sequence):
```

Comment:
Was moving this into the previous if block intentional?  I think it is still correct this way, but just checking.